hyperledger-student-management.yaml

```diff
@@ -0,0 +1,232 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Student API for UC4's hyperledger component."
+  version: "0.3.3"
+  title: "UC4"
+servers:
+  - url: http://localhost:9000/hyperledger-student-management
+tags:
+- name: "Student Management"
+  description: "Everything about the student"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+paths:
+  /users/students:
+    post:
```

Comment:
I'd assume you send the existing proper format to lagom and they proxy for you, but we should definitely talk about this.

Reply:
It would be ludacris to habe two endpoints which the exact same function.
You are getting one endpoint which looks exactly more or less the same as the current.